app.rb

```diff
@@ -0,0 +1,161 @@
+require_relative 'book'
+require_relative 'teacher'
+require_relative 'student'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def run
+    puts 'Welcome to School Library App!'
+    menu
+  end
+
+  def menu
+    loop do
+      puts '' * 10
+      puts 'Please choose an option by entering a number:'
+      puts '1 - List all books'
+      puts '2 - List all people'
+      puts '3 - Create a person'
+      puts '4 - Create a book'
+      puts '5 - Create a rental'
+      puts '6 - List all rentals for a given person id'
+      puts '7 - Exit'
+      print ''
+
+      choice = gets.chomp.to_i
+
+      case choice
+      when 1..6 then options(choice)
+      when 7 then puts 'Thank you for using our app.'
+      end
+      break if choice == 7
+    end
+  end
+
+  def options(choice)
+    case choice
+    when 1
+      list_books
+    when 2
+      list_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
```

Comment:
- [ ] Kindly take note that I was unable to create a rental succesfully, I got the error messge screenshot below 👇 

<img width="581" alt="Untitled" src="https://github.com/Mofuhidy/OOP-school-library/assets/103386183/1ac65b4b-c118-4bd7-8acc-6768a99deb68">